src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -223,10 +223,10 @@ void BaseQNAcceleration::updateDifferenceMatrices(
         residualMagnitude /= utils::IntraComm::l2norm(_values);
       }
 
-      PRECICE_CHECK(not math::equals(residualMagnitude, 0.0),
-                    "Attempting to add a zero vector to the quasi-Newton V matrix. This means that the residuals "
-                    "in two consecutive iterations are identical. If a relative convergence limit was selected, "
-                    "consider increasing the convergence threshold.");
+      if (math::equals(residualMagnitude, 0.0)) {
+        PRECICE_WARN("Add a zero vector to the quasi-Newton V matrix. This means that the residuals "
+                     "in two consecutive iterations are identical.");
+      }
```

Comment:
```suggestion
      if (math::equals(residualMagnitude, 0.0)) {
        PRECICE_WARN("Adding a vector with a two-norm of {} to the quasi-Newton V matrix, which will lead to "
                     "ill-conditioning. A filter might delete the column again. Still, this could mean that you are " 
                     "converging too tightly, that you reached steady-state, or that you are giving by mistake identical "
                     "data to preCICE in two consecutive iterations.", residualMagnitude);
      }
```

having the exact norm value could help debugging ... is it 1e-15 or really 0.